setup.py

```diff
@@ -0,0 +1,28 @@
+import setuptools
+import re
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+with open('requirements.txt') as f:
+    requirements = f.read().splitlines()
+print(requirements)
```

Comment:
I think we can remove the printing of requirements, or if we want to keep it, maybe add some more text to make it clear what is being printed to screen.